polars/polars-core/src/chunked_array/logical/categorical/ops/unique.rs

```diff
@@ -42,6 +42,10 @@ impl CategoricalChunked {
         }
     }
 
+    pub fn approx_n_unique(&self, precision: u8) -> PolarsResult<usize> {
```

Comment:
I don't think we should implement this on the logical types. I shall explain how we should do this in the review summary.

Reply:
done.